src/cli/auth.js

```diff
@@ -0,0 +1,119 @@
+import fs from 'fs'
+import path from 'path'
+import prompt from 'prompt'
+import chalk from 'chalk'
+import validUrl from 'valid-url'
+import axios from 'axios'
+import Confirm from 'prompt-confirm'
+
+import util from '../util'
+
+const AUTH_FILE = '.auth.json'
+
+// TODO: this part replicates `start.js`, refactor
+const getDataDir = () => {
+  const projectPath = path.resolve('.')
+
+  const botfile = path.join(projectPath, 'botfile.js')
+  if (!fs.existsSync(botfile)) {
+    util.print('error', `(fatal) No ${chalk.bold('botfile.js')} file found at: ` + botfile)
+    process.exit(1)
+  }
+
+  // eslint-disable-next-line no-eval
+  const bf = eval('require')(botfile)
+  return util.getDataLocation(bf.dataDir, projectPath)
+}
+
+const getAuthFile = () => path.join(getDataDir(), AUTH_FILE)
+
+const readAuth = () => {
+  const authFile = getAuthFile()
+  try {
+    const json = fs.readFileSync(authFile, 'utf-8')
+    return JSON.parse(json)
+  } catch (err) {
+    if (err.code !== 'ENOENT') {
+      console.warn(`Error reading ${authFile}:`, err.message)
+    }
+  }
+  return {}
+}
+
+const writeAuth = auth => {
+  const authFile = getAuthFile()
+  fs.writeFileSync(authFile, JSON.stringify(auth, null, 2))
+}
+
+exports.login = botUrl => {
```

Comment:
@emirotin , can we use ES6 modules syntax here and for `logout`?

Reply:
We seem to be using CJS exports everywhere, I'm not sure why, but I've kept the style of the rest of the codebase. Happy to change though if you say it's safe, but we will have to later change the rest to keep the code consistent.

Reply:
I'd also vote for using ES6 syntax everywhere where it's possible.